gammapy/scripts/cta_sensitivity.py

```diff
@@ -0,0 +1,250 @@
+#-*- coding: utf-8 -*-
+
+"""
+Compute the CTA sensitivity for a fixed observation config
+Authors: J. Lefaucheur (LUTh), B. Khelifi (APC)
+Date: 31/05/2017
+Version: 0.1
+
+Note: For the moment, only the differential point-like sensitivity is computed
+ToDo: - make the computation for any source size
+      - compute the integral sensitivity
+      - Add options to use different spectral shape?
+
+Inputs:
+    - filename: str
+        IRF filename (full path, in general there are here: ../datasets/cta/)
+        e.g. ../datasets/cta/CTA-Performance-South-20170323/CTA-Performance-South-20deg-30m_20170323.fits
+    - livetime: u.h
+        Livetime (object with the units of time), e.g. 5*u.h
+Optionnal inputs:
+    - slope: float (>0)
+        Index of the spectral shape (Power-law)
+    - alpha: float
+        On/OFF normalisation
+    - sigma: float
+        minimun significance
+    - gamma_min: float
+        Minimum number of gamma-rays
+    - bkg_sys: float
+        Fraction of Background systematics relative to the number of ON counts
+    - verbosity: int
+        level of print out [int: 0/1/2(debug)]. In the debug mode, stats for each bin are printed and the computed \
+        sensitivity curve is superposed with the one stored in the IRF fits file
+
+"""
+
+# Inputs :
+
+# IRFs :
+#  - taux de fond en énergie reconstruite
+#  - matrice de dispersion en énergie
+#  - surface efficace en énergie vraie
+# Paramètres :
+#  - nsigma, alpha, temps, ngamma_min et systématiques fond
+
+# Taux de fond, calcul de nombre de gamma
+
+# calcul du flux avec la double intégrale et le nombre de gamma
+
+import astropy.units as u
+import numpy as np
+import matplotlib.pyplot as plt
+import sys
+
+from gammapy.stats import significance_on_off
+from gammapy.spectrum.models import PowerLaw
+from gammapy.spectrum.utils import CountsPredictor
+from gammapy.scripts import CTAPerf
+from gammapy.utils.scripts import get_parser, set_up_logging_from_args
+
+
+class cta_sensi_estim(object):
```

Comment:
There's a convention in Python (the PEP8 coding style guide) that class names are `CamelCase`, and module / variable / function / method names are `snake_case`.
So here it should be `CTASensiEstim`, not `cta_sensi_estim`.

Actually, I would suggest to call this class `SensitivityEstimator` or `SensitivityEstimator1D` or something like that. Really it should work for any instrument (we have to change names in other places as well, e.g. the `CTAPerf` class), there's nothing CTA-specific here.